src/Wallabag/ApiBundle/Controller/EntryRestController.php

```diff
@@ -564,7 +564,7 @@ public function patchEntriesReloadAction(Entry $entry)
         }
 
         // if refreshing entry failed, don't save it
-        if ($this->getParameter('wallabag_core.fetching_error_message') === $entry->getContent()) {
+        if ($this->container->getParameter('wallabag_core.fetching_error_message') === $entry->getContent()) {
```

Comment:
If we start make cleaner we might as well inject parameters properly inside `services.yml`.

Reply:
But that's for another PR…

Reply:
We should inject service into controller too, etc. :)

Reply:
But thah change was mostly because the parent is now extending `AbstractFOSRestController` instead of `FOSRestController` (which force us to use `$this->container` for parameter because the controller wasn't "container aware")